packages/contact-service/jest.config.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+export default {
+  displayName: 'contact-service',
+  testEnvironment: 'node',
+  preset: 'ts-jest',
+  transform: {
+    '^.+\\.[tj]s$': ['ts-jest', { tsconfig: '<rootDir>/tsconfig.spec.json' }],
+  },
+  transformIgnorePatterns: ['/node_modules/(?!(\\@adorsys-gis/event-bus)/)'],
+  moduleFileExtensions: ['ts', 'js', 'html'],
```

Comment:
Why treat the event bus library differently? Please double check that its specified main file is typescript.